apps/bfd-server/dev/api-changelog.md

```diff
@@ -1,5 +1,65 @@
 # API Changelog
 
+## BFD-1620: Add Total Slices for CARIN Compliance
+
+Adds two slices for C4BBAdjudication and C4BBPayerAdjudicationStatus to be in compliance with CARIN.
+
+C4BBAdjudication example:
+
+C4BBPayerAdjudicationStatus example:
```

Comment:
I think line 9 is not supposed to be here?

Reply:
Good catch, fixed!